pkg/ui/src/views/cluster/containers/dataDistribution/replicaMatrix.tsx

```diff
@@ -94,6 +100,32 @@ class ReplicaMatrix extends Component<ReplicaMatrixProps, ReplicaMatrixState> {
     return `${arrow} ${label}`;
   }
 
+  rowLabel(row: FlattenedNode<SchemaObject>) {
+    const text = this.rowLabelText(row);
+
+    const droppedIndicator = row.data.droppedAt
```

Comment:
Since you've got the conditional below anyway, you can get rid of this one entirely.

Reply:
Got rid of this anyway since changed from asterisk to greying out dropped tables